sdk/search/Azure.Search.Documents/README.md

```diff
@@ -269,6 +269,7 @@ SearchOptions options = new SearchOptions
     Filter = SearchFilter.Create($"Rating ge {stars}"),
     Size = 5, // Take only 5 results
     OrderBy = { "Rating desc" } // Sort by Rating from high to low
+    Select = {"HotelName", "Rating"} // Include only the columns that you are interested in. By default all columns are included
```

Comment:
This may serve better as a separate sample. Samples in the README are meant to be the common scenarios, and selecting a single field is probably not very common if you're intent is to display results.

You can create a separate file under the _sdk/search/Azure.Search.Documents/tests/samples_ directory. A _.cs_ is needed along with a `#region` around it (see other samples for an exmaple), then a markdown file under _sdk/search/Azure.Search.Documents/samples_ with that references that code snippet.